integrations/vello_svg/src/lib.rs

```diff
@@ -33,78 +33,107 @@
 //! - patterns
 
 use std::convert::Infallible;
-
 use usvg::NodeExt;
-use vello::kurbo::{Affine, BezPath, Rect, Stroke};
-use vello::peniko::{Brush, Color, Fill};
-use vello::SceneBuilder;
+use vello::{
+    kurbo::{Affine, BezPath, Point, Rect, Stroke},
+    peniko::{Brush, Color, Fill},
+    SceneBuilder,
+};
+
+/// Re-export vello.
+pub use vello;
 
+/// Re-export usvg.
 pub use usvg;
 
-/// Append a [`usvg::Tree`] into a Vello [`SceneBuilder`], with default error handling
-/// This will draw a red box over (some) unsupported elements
+/// Append a [`usvg::Tree`] into a Vello [`SceneBuilder`], with default error
+/// handling This will draw a red box over (some) unsupported elements
```

Comment:
This is an odd change.

I must have failed to put two spaces after `handling`, previously. I'd propose at least putting a full stop between the two

Reply:
Will be fixed. It was a bad auto-format.